### PR TITLE
updated hub programming instructions in README (CU-vf96g1)

### PR DESCRIPTION
**Pull request by schwarrrtz**
pretty self-explanitory.

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-vf96g1 review and finalize documentation for Ansible and the config repo](https://app.clickup.com/t/vf96g1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ the code was deployed.
 ### Added
 
 - Sound effects to push notifications (CU-10xfkhr).
+- Additional hub programming instructions to the README.
```
Comment (trisapeace) [inline CHANGELOG.md:17]: Can you please add the ClickUp number?

File: README.md [hunk 2]
```diff
@@ -93,6 +97,10 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+1. ssh into the RPi, copy the system_id from `\usr\local\brave\system_id` and add it to the Button Config Google Sheet
+
+1. add an entry for the new hub to the `hubs` table in the db
+
 # How to deploy updates to the RPi fleet using Ansible
```
Comment (trisapeace) [inline README.md:104]: Somewhere in the README, could you please also include instructions on how to limit the command to just one RPi?

**Review by trisapeace — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (schwarrrtz) [reply CHANGELOG.md]: done.

File: README.md (hunk 2 above)
Comment (schwarrrtz) [reply README.md:104]: This pull request isn't the full updated Ansible documentation - I've added your request to that task on ClickUp so I don't forget.

**Pull request merged**